analysis/technical-analysis.md

```diff
@@ -0,0 +1,474 @@
+the host, the host operating system and hypervisor, system administrators, service providers, and the
+infrastructure owner—or anyone else with physical access to the hardware.
+
+Together, these attributes provide not only an assurance that the data is kept confidential, but also that
+the computations performed are actually the correct computations, allowing one to trust the results of the
+computation as well.
+
+Depending on the particulars of a specific TEE, it may also provide:
+
+* **Code Confidentiality**: In addition to protecting data, some TEEs may protect code while in use from
+being viewed by unauthorized entities. For example, this can protect an algorithm that is considered to
+be sensitive intellectual property.
+* **Authenticated Launch**: Some TEEs may enforce authorization or authentication checks prior to
+launching a requested process and may refuse to launch a process that is not authorized or authenticated.
+* **Programmability**: Some TEEs may be programmed with arbitrary code, while some may only support
+a limited set of operations. A TEE might even include or be composed entirely of code fixed at the time
+of manufacture.
+* **Attestability**: Often, a TEE can provide evidence or measurements of its origin and current state,
+so that the evidence can be verified by another party and—programmatically or manually—it can
+decide whether to trust code running in the TEE. It is typically important that such evidence is signed
+by hardware that can be vouched for by a manufacturer, so that the party checking the evidence
+has strong assurances that it was not generated by malware or other unauthorized parties. (Further
+discussion on attestation is provided later in section 7.)
+* **Recoverability**: Some TEEs may provide a mechanism for recovery from a non-compliant or potentially
+compromised state. For example, if it is determined that a firmware or software component no longer
+meets compliance requirements and the launch authentication mechanism fails, it may be possible
+to update that component and retry (recover) the launch. Recoverability generally requires that
+some component(s) of the TEE remain trusted, which can act as a “root” when other components are
+updated. (Further discussion on such recovery is provided later in section 7.)
+
+A hardware-based TEE uses hardware-backed techniques to provide increased security guarantees for
+the execution of code and protection of data within that environment. This assurance is often missing in
+approaches that do not use a hardware-based TEE.
+
+For more discussion of TEE definitions used in the industry, see the discussion at [6].
+
+### 4 Related Technologies
+
+The TAC conducted a survey of various terms in the industry related to protecting data in use, and
+composed the following Venn diagram of technologies:
+
+![Alt text](related-terminology.png "Related Technologies")
```

Comment:
This will not render correctly since the PNG file is not in the folder.

(The name of the PNG should probably be `related-technologies.png`.)